user_sync/connector/directory_ldap.py

```diff
@@ -137,6 +138,7 @@ def get_options(caller_config):
         builder.set_string_value('user_given_name_format', six.text_type('{givenName}'))
         builder.set_string_value('user_surname_format', six.text_type('{sn}'))
         builder.set_string_value('user_country_code_format', six.text_type('{c}'))
+        builder.set_string_value('user_memberof_format', six.text_type('{memberOf}'))
```

Comment:
Since we aren't making this option formattable, we should remove the curly braces from the default value. Let's also change the default value to `'member'` since that is the default for the two-step attribute (which we will need to reconcile with this new feature).

Reply:
On second thought, the default value should be `None` since this must be optional for basic use cases (e.g. when Dynamic group mapping and two-step lookup are not in use).